src/Tribe/Adjacent_Events.php

```diff
@@ -0,0 +1,329 @@
+<?php
+/**
+ * Controls getting a previous or next event from the context of a single event being viewed.
+ */
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
Also not necessary.

Reply:
many of our files have this, so was just copying their approach — any reason for why some have it and some don't?

Reply:
This was a copied style from WP Core, that doesn't accomplish anything for us.